source/index.js

```diff
@@ -36,15 +37,32 @@ var likely = function (node, options) {
 };
 
 /**
+ * Will be deprecated in 3.0
+ * @returns {Likely}
+ */
+var likely = function () {
+    // eslint-disable-next-line no-console
+    console.warn('This function will be DEPRECATED in 3.0. Use likely.initiate instead.');
+    return likely.initiate.apply(likely, arguments);
+};
+
+/**
  * Initiate Likely buttons on load
+ * @param {Node|Object} [node] a particular node of widget,
+ *                             if not specified,
+ *                             tries to init all the widgets
  * @param {Object} [options] additional options for each widget
  */
-likely.initiate = likely.initate = function (options) {
-    var widgets = dom.findAll('.' + config.name);
+likely.initiate = likely.initate = function (node, options) {
```

Comment:
What if we also make `initiate()` accept an array of nodes? That’ll make possible calling `likely.initiate([node1, node2])` instead of `likely.initiate(node1); likely.initiate(node2);`. It’s a valid use-case for Tinkoff Journal, for example.